src/modules/compliance/src/lib/Base64.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <string>
+#include "Result.h"
+#include "Base64.h"
+
+namespace compliance
+{
+    static inline char Base64Char(const unsigned char c)
+    {
+        if (c >= 'A' && c <= 'Z')
+        {
+            return c - 'A';
+        }
+        else if (c >= 'a' && c <= 'z')
+        {
+            return c - 'a' + 26;
+        }
+        else if (c >= '0' && c <= '9')
+        {
+            return c - '0' + 52;
+        }
+        else if (c == '+')
+        {
+            return 62;
+        }
+        else if (c == '/')
+        {
+            return 63;
+        }
+        else
+        {
+            return 0;
+        }
+    }
+
+    static inline bool is_base64(const unsigned char c)
```

Comment:
>is_base64 [](http://example.com/codeflow?start=23&length=9)

Please use correct casing for names Function names should be PascalCased. Thus, this would be: 

IsBase64()

Here and elsewhere.